python/build_template/configure_build.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/python3
+from lib import ninja_syntax
+from pathlib import Path
+import sys
+import argparse
+import os
+import dx_build
+
+game_name = "Rock Band 3 Deluxe"
+
+ninja = ninja_syntax.Writer(open("build.ninja", "w+"))
+
+#versions
+#gh1, gh2 - " "
+#rb1, track packs - "-v 4"
+#rb2, tbrb, gdrb - "-v 5"
+#rb3, dc, blitz - "-v 6"
+ninja.variable("ark_version", "-v 6")
+ark_version = "-v 6"
```

Comment:
if this is gonna be a ninja var you should remove the constant

Reply:
idk the best convention to allow manipulating the variable for the edge case before applying it

Reply:
OH
no i just forgot to delete the constant my bad